--- a/docs/conf.py
+++ b/docs/conf.py
@@ -176,7 +176,7 @@ towncrier_draft_include_empty = False
 
 # Add icons (aka "favicons") to documentation
 sys.path.append(os.path.join(os.path.dirname(__file__), '_ext'))
-extensions += ['_custom_icons']
+#extensions += ['_custom_icons']
 
 # List of dicts with <link> HTML attributes
 # as defined in https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link
